src/main/java/svc/dto/MunicipalitiesDTO.java

```diff
@@ -0,0 +1,13 @@
+package svc.dto;
+
+import java.util.List;
+
+import svc.models.Municipality;
+
+public class MunicipalitiesDTO  {
```

Comment:
Maybe instead of a new object that contains a list, make the object extend List itself:
i.e.
public class Municipalities extends List<Municipality> { }